### PR TITLE
consumerid=distributor id

### PR DESCRIPTION
**Pull request by vritant**
* Doesnt allow using --force with --consumerid
* Stopped --consumerid = distributor id

### CODE REVIEW

**Comment by wottop** [comment]
retest this please

**Comment by wottop** [comment]
make stylish run:
test/test_registration.py:104:83: E231 missing whitespace after ','
test/test_registration.py:122:83: E231 missing whitespace after ','
Makefile:646: recipe for target 'flake8' failed
make: *** [flake8] Error 1

**Comment by vritant** [comment]
@wottop sorry did not know we had this. I fixed the issues , merged master and added a pre-commit hook in my repo so this doesnt happen again.

**Comment by vritant** [comment]
retest this please

File: src/subscription_manager/managercli.py [hunk 1]
```diff
@@ -1057,6 +1059,10 @@ def _do_command(self):
                         self.options.consumerid)
                 consumer = admin_cp.getConsumer(self.options.consumerid,
                         self.username, self.password)
+                if consumer['type']['manifest']:
+                    log.error("registration attempted with consumerid = Subscription Management Application's uuid: %s" % self.options.consumerid)
+                    system_exit(os.EX_USAGE, _("Error: Cannot register with ID of a Subscription Management Application: %s" % self.options.consumerid))
```
Comment (wottop) [inline src/subscription_manager/managercli.py]: with *an* ID

**Comment by wottop** [comment]
One fix to the error message, then I will ACK.

Do not put this on the cherry-pick list for 7.2. It would require a string change and we are at string freeze.

**Comment by wottop** [comment]
ack

**Pull request merged**